### PR TITLE
Implement Milking Predictions

### LINKED ISSUE #285: Provide predictions for expected milk yield
Hi, 

Could TestDayResultEventResource have a new property:   "milkWeightExpected" or "milkWeightGoal" ? 

BR

### PR DESCRIPTION
**Pull request by erwinspeybroeck**

### CODE REVIEW

File: url-schemes/milkURLScheme.json [hunk 1]
```diff
@@ -376,6 +376,45 @@
                 }
             }
         },
+        "/locations/{location-scheme}/{location-id}/milk-predictions": {
+            "get": {
+                "operationId": "get-milk-predictions",
+                "summary": "Get the milk predictions for a certain location",
+                "description": "# Purpose\nProvides the milk predictions for a location\n",
+                "tags": [
+                    "ADE-1.4-milk"
```
Comment (erwinspeybroeck) [inline url-schemes/milkURLScheme.json]:
```suggestion
                    "ADE-1.3-milk"
```

File: types/icarMilkingPredictionType.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "description": "The amount of milk, fat and protein milked in a defined period of time",
+
+  "type": "object",
+
+  "required": [
+    "unitCode",
```
Comment (AlexeyHardCode) [inline types/icarMilkingPredictionType.json]: Are these two properties were accidentally added? or it should be like this?

File: resources/icarTestDayResultEventResource.json [hunk 3]
```diff
@@ -19,7 +19,10 @@
           "items": {
             "$ref": "../types/icarMilkCharacteristicsType.json"
           }
-        }
+        },
+        "predictedProductionOnTestday": {
```
Comment (AlexeyHardCode) [inline resources/icarTestDayResultEventResource.json]: Should the word "Day" be started from capital letter? 
"predictedProductionOnTestDay"

Comment (erwinspeybroeck) [inline resources/icarTestDayResultEventResource.json]:
```suggestion
        "predictedProductionOnTestDay": {
```

File: types/icarMilkingPredictionType.json (hunk 2 above)
Comment (erwinspeybroeck) [inline types/icarMilkingPredictionType.json]:
```suggestion
    "milkWeight"
```

File: types/icarMilkingPredictionType.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    "value"
```
Comment (erwinspeybroeck) [inline types/icarMilkingPredictionType.json]:
```suggestion
```

**Review by cookeac — APPROVED** [review]
Looks good now.

**Comment by cookeac** [comment]
Resolves #285

**Comment by cookeac** [comment]
I think we've given this plenty of time for review and have discussed at previous meetings. Merging now.

**Pull request merged**